database/engine/journalfile.c

```diff
@@ -13,7 +13,7 @@ static void flush_transaction_buffer_cb(uv_fs_t* req)
 
     uv_fs_req_cleanup(req);
     free(io_descr->buf);
```

Comment:
why not freez here too? I assume combined with using the respective mallocz too (haven't checked the declaration)

Reply:
there is no mallocz here.